app/models/user.js

```diff
@@ -27,5 +32,20 @@ module.exports = (sequelize, DataTypes) => {
     }
   );
 
+  User.createModel = user =>
+    User.create(user)
+      .then(success => {
+        logger.info(`User ${success.dataValues.firstName} created correctly.`);
+      })
+      .catch(err => {
+        logger.info(`${user.firstName} user no created.`);
+        logger.error(err);
+        if (err.name === 'SequelizeUniqueConstraintError') {
+          const signErrors = [];
```

Comment:
This constant is unneeded, you can declare an array like `['User already exist']`